package.json

```diff
@@ -199,6 +202,7 @@
     "react-dnd": "^15.1.2",
     "react-dnd-html5-backend": "^15.1.3",
     "react-is": "^17.0.2",
+    "react-markdown": "^8.0.7",
```

Comment:
**question**: as this is only used for storybook, could this be a dev dependency?